CHANGELOG.md

```diff
@@ -1,3 +1,463 @@
+
+### Elevation
+
+#### UIView(MaterialElevationResponding)
+
+*new* method: `-mdc_elevationDidChange` in `UIView(MaterialElevationResponding)`
+
+*new* property: `mdc_absoluteElevation` in `UIView(MaterialElevationResponding)`
+
+#### UIColor(MaterialElevation)
+
+*new* method: `-mdc_resolvedColorWithElevation:` in `UIColor(MaterialElevation)`
+
+*new* method: `-mdc_resolvedColorWithTraitCollection:elevation:` in `UIColor(MaterialElevation)`
+
+*new* category: `UIColor(MaterialElevation)`
+
+### FeatureHighlight
+
+#### MDCFeatureHighlightViewController
+
+*new* property: `traitCollectionDidChangeBlock` in `MDCFeatureHighlightViewController`
+
+#### MDCFeatureHighlightView
+
+*new* property: `traitCollectionDidChangeBlock` in `MDCFeatureHighlightView`
+
+*modified* class: `MDCFeatureHighlightView`
+
+| Type of change: | Declaration |
+|---|---|
+| From: | `@interface MDCFeatureHighlightView : UIView  @property(nonatomic, strong, nullable) UIColor *innerHighlightColor UI_APPEARANCE_SELECTOR; @property(nonatomic, strong, nullable) UIColor *outerHighlightColor UI_APPEARANCE_SELECTOR;  @property(nonatomic, strong, nullable) UIFont *titleFont UI_APPEARANCE_SELECTOR; @property(nonatomic, strong, nullable) UIColor *titleColor UI_APPEARANCE_SELECTOR;  @property(nonatomic, strong, nullable) UIFont *bodyFont UI_APPEARANCE_SELECTOR; @property(nonatomic, strong, nullable) UIColor *bodyColor UI_APPEARANCE_SELECTOR;  /*  Indicates whether the view's contents should automatically update their font when the device’s  UIContentSizeCategory changes.   This property is modeled after the adjustsFontForContentSizeCategory property in the  UIContentSizeCategoryAdjusting protocol added by Apple in iOS 10.0.   Default value is NO.  */ @property(nonatomic, readwrite, setter=mdc_setAdjustsFontForContentSizeCategory:)     BOOL mdc_adjustsFontForContentSizeCategory UI_APPEARANCE_SELECTOR;  /**  Enable legacy font scaling curves for Dynamic Type  Default value is NO.  */ @property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:) BOOL mdc_legacyFontScaling;  @end` |
```

Comment:
This seems wrong???

Reply:
I have no idea how apidiff works. But it always seems wrong to me.